File: MinecraftPvpPlugin/src/main/java/me/minecraft/minecraftpvpplugin/skills/Teleport.java
package me.minecraft.minecraftpvpplugin.skills;

import me.minecraft.minecraftpvpplugin.Skill;
import org.bukkit.Material;
import org.bukkit.entity.Player;
import org.bukkit.event.player.PlayerInteractEvent;

public class Teleport extends Skill{
    public Teleport(Material material, String name) {
        super(material, name);
    }

    @Override
    public void onActivate(PlayerInteractEvent event) {
        Player player = event.getPlayer();
    }
}

File: MinecraftPvpPlugin/src/main/java/me/minecraft/minecraftpvpplugin/skills/Teleport.kt
package me.minecraft.minecraftpvpplugin.skills

import me.minecraft.minecraftpvpplugin.Skill
import org.bukkit.Material
import org.bukkit.event.player.PlayerInteractEvent

class Teleport(material: Material?, name: String?) : Skill(material, name) {
    public override fun onActivate(event: PlayerInteractEvent) {
        val player = event.player
    }
}
